### PR TITLE
[shopsys] faq: added tip for emails debugging

### PR DESCRIPTION
**Pull request by vitek-rostislav**
| Q             | A
| ------------- | ---
|Description, reason for the PR| a tip from @adamtyron :+1: There is a nice tool that can be used for debugging emails when using Docker - thanks to the tool, you can see all the emails sent by the application (including their headers) in a web UI
|New feature| No 
|[BC breaks](https://github.com/shopsys/shopsys/blob/master/docs/contributing/backward-compatibility-promise.md)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Comment by vitek-rostislav** [comment]
Hi @TomasLudvik, I have changed it - we now use the new image by default :wink:

**Comment by PetrHeinz** [comment]
So now localhost will never send any e-mails by default? That might be counter-intuitive... Is that OK, @LukasHeinz

**Comment by vitek-rostislav** [comment]
oh, @PetrHeinz, thanks for pointing this out, I did not think about it this way :disappointed: :man_facepalming:

**Comment by PetrHeinz** [comment]
Also, you've regressed to limiting the IP address in exposing the ports. I've requested to change that in sake of consistency in https://github.com/shopsys/shopsys/pull/1252#discussion_r307807613.

**Comment by vitek-rostislav** [comment]
ok, after this discussion, I will change this PR back to the original state - just a tip in the docs...

**Comment by PetrHeinz** [comment]
@LukasHeinz Well, I've asked about whether the change Rosťa did (stop sending e-mails in dev by default and include `djfarrelly/maildev` instead) is OK with you.

image

Rosťa thought about it and after a while of silent contemplation reverted the changes we've discussed.
image

**Comment by vitek-rostislav** [comment]
Hi guys, the description updated, thanks for noticing :+1:

**Comment by TomasLudvik** [comment]
Hello @vitek-rostislav, so that docker image stops sending e-mails and only displays them? I thought it sends e-mail as previous image and displays all images sent in the UI. If this is not true:
- you should at least describe it in docs
- you could try to find image (or maybe just change some settings) to provide full functionality.

**Comment by PetrHeinz** [comment]
@TomasLudvik Wait, you did the testing - you must know whether the e-mail were actually sent.

I've just assumed from the documentation - the image can relay the mail to another SMTP server, but that wasn't configured in the `docker-compose.yml`. That's why I've asked about the behavior in the first place in https://github.com/shopsys/shopsys/pull/1252#issuecomment-517288001:
> So now localhost will never send any e-mails by default?

**Comment by PetrHeinz** [comment]
OK, so I've just tried out the `djfarrelly/maildev` image myself to be completely sure that it does not send any e-mails in the default configuration. Their docs include [Outgoing emails](https://github.com/djfarrelly/MailDev#outgoing-email) section that explains the possibilities.

**Comment by TomasLudvik** [comment]
@vitek-rostislav I think that best way would be to add this as new service and relay e-mails via our current SMTP.

File: docs/introduction/faq-and-common-issues.md [hunk 1]
```diff
@@ -125,4 +126,21 @@ Otherwise, the logic is in facades (resp. the facades are used as delegates to o
...
+    ```diff
+     smtp-server:
+   -        image: namshi/smtp:latest
+   +        image: djfarrelly/maildev
+             container_name: shopsys-framework-smtp-server
+   +        ports:
+   +            - "8025:80"
+    ```
+1. Run `docker-compose up -d`
+1. Now you are able to see all the application emails in the inbox on `http://127.0.0.1:8025`
```
Comment (DavidOstrozlik) [inline docs/introduction/faq-and-common-issues.md]: one last thing :D please make `http://127.0.0.1:8025` a link

Comment (vitek-rostislav) [reply docs/introduction/faq-and-common-issues.md]: fixed in https://github.com/shopsys/shopsys/pull/1252/commits/02c16b9f072be168eee4b7a712fdaa7d3f7cfd53

**Pull request merged**